CMakeLists.txt

```diff
@@ -250,13 +249,7 @@ if (ENABLE_COMPILER_WARNINGS)
 endif()
 
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
-if (PARTHENON_ENABLE_CPP17)
-  set(CMAKE_CXX_STANDARD 17)
-# Ensure correct flag is used as deprecated c++1z is not supported by nvcc.
-  set(CMAKE_CXX17_STANDARD_COMPILE_OPTION "-std=c++17")
```

Comment:
Is this no longer required?

Reply:
IIRC the correct flags are now provided by Kokkos

Reply:
Just confirmed that I can compile with cuda on my machine without that line.